### ISSUE TITLE
Time varying effects plotting error

### ISSUE THREAD

**Issue by lpxhonneux** [drizopoulos/JMbayes#32]
Thank you for the JMBayes package.

When attempting to plot the time-varying effects using 'plot(model, which = "tv_effect")' we receive an error. Upon further inspection, we realised this is caused by the variable 'td_cols' (which indicates the coefficients of the time-varying effect spline) also referred to the additional interaction term (antibody_presence).

pseudocode:

Interactions <- list("signature" ~ 0 + tve(endptageyear, df = 8) + antibody_presence)

For the example above td_cols would be

[[1]]
    [1] 1 2 3 4 5 6 7 8 9 10 11

where coefficients 1:10 refer to the tve spline and 11 refers to the coefficient of antibody-presence. This affects the computation of the model during mcmc sampling and hence we were wondering whether the inclusion of the additional coefficient (in this case #11, 'antibody presence') is intentional or a programming bug? 

In other words, in the example above, is it intended that 'td_cols' would read as below, with #11 omitted?

[[1]]
    [1] 1 2 3 4 5 6 7 8 9 10

If this is the case, would a possible solution be to alter the function in mvJointModelBayes from:

td_cols <- mapply(seq, from = c(1, head(cumsum(sapply(U, ncol)), -1) + 1),
                     to = cumsum(sapply(U, ncol)), SIMPLIFY = FALSE)[which_td]

to:

tmp <- mapply(seq, from = c(1, head(cumsum(sapply(U, ncol)), -1) + 1),
                      to = cumsum(sapply(U, ncol)), SIMPLIFY = FALSE)
td_cols <- mapply(function (a,b) {b[a]}, a = td_cols, b = tmp, SIMPLIFY = FALSE)
